images/router/haproxy/reload-haproxy

```diff
@@ -12,8 +12,13 @@ if [ -f $pid_file ]; then
   old_pid=$(<$pid_file)
 fi
 
+# Assumes that the docker ip is the first entry in the hosts file.
+# This port is non-configurable via the router cmd, it is HAProxy specific.  It should
+# match the port used in the haproxy-config.template
+peer_name=$(awk 'NR==1 {print $1}' /etc/hosts):1937
```

Comment:
Actually, I'm going to test one more thing before I try and hack something up.  If I can put a static entry for `peer self 0.0.0.0:1937` in the list without it puking due to a "duplicate" with the endpoint ip then we're fine.